.devcontainer/postStart.sh

```
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Check on couchdb
+USR=admin
```

Shellcheck (0.10.0)

[warning] Use var=$(command) to assign output (or quote to assign string).

(SC2209)